test/module/PasskeyFlow.module.specs.ts

```diff
@@ -92,21 +90,5 @@ describe("Passkeys Registry Module:", function () {
       "nonce"
     );
     userOp1.signature = "";
-    console.log("userOp1 before", userOp1);
-    // add validator module address to the signature
-    // const signatureWithModuleAddress =
-    //   "0x26877a3a3c8313f8a23dd5323d74523613e8ed12581731be9df482b969efc6b0917f457dfe4fea9bffa6323a29e9ca56bebcec612129f76883690434ae168e3d1d9dbfe496790f59daeeb62c5c8b00fc5a9751c9caa76b194e5472b190b8690100000000000000000000000000000000000000000000000000000000000000c000000000000000000000000000000000000000000000000000000000000001200000000000000000000000000000000000000000000000000000000000000180000000000000000000000000000000000000000000000000000000000000002549960de5880e8c687434170f6476605b8fe4aeb9a28632c7995cf3ba831d9763050000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000247b2274797065223a22776562617574686e2e676574222c226368616c6c656e6765223a22000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000037222c226f726967696e223a22687474703a2f2f6c6f63616c686f73743a35313733222c2263726f73734f726967696e223a66616c73657d000000000000000000";
-
-    // userOp1.signature = signatureWithModuleAddress;
-    // console.log("userOp1", userOp1);
-
-    // const handleOpsTxn = await entryPoint.handleOps(
-    //   [userOp1],
-    //   await offchainSigner.getAddress(),
-    //   {
-    //     gasLimit: 20000000,
-    //   }
-    // );
-    // await handleOpsTxn.wait();
   });
 });
```

Comment:
"Can send a userOp with a default validation module" test case is not clear or incomplete in this file.